src/target/source/codegen_cuda.cc

```diff
@@ -1197,8 +1197,10 @@ inline void PrintConst(const FloatImmNode* op, std::ostream& os, CodeGenCUDA* p)
       break;
     }
     case 16: {
-      os << "__float2half_rn";
-      os << '(' << std::scientific << op->value << 'f' << ')';
+      os << "__float2half_rn" << '(';
```

Comment:
Would that be possible that we can add some test? We can probably re-use my repro [here](https://gist.github.com/shingjan/9ee06597a13f9be324828afb6c4560b8)

Reply:
the case is added